hardware_tests/test_polarization_controller_thorlabs_mpc320.py

```diff
@@ -75,6 +75,25 @@ def test_move_absolute(device: PolarizationControllerThorlabsMPC320) -> None:
     # device.move_absolute(ChanIdent.CHANNEL_3, 0 * ureg.degree)
 
 
+def test_jog(device: PolarizationControllerThorlabsMPC320) -> None:
+
+    jog_step_1 = 50
+    jog_count_1 = 5
+
+    device.set_params(jog_step_1=jog_step_1, jog_step_2=25, jog_step_3=25)
+
+    device.home(ChanIdent.CHANNEL_1)
+    device.home(ChanIdent.CHANNEL_2)
+    device.home(ChanIdent.CHANNEL_3)
+
+    # Compare jog distance versus absolute distance
+    for _ in range(jog_count_1):
+        device.jog(ChanIdent.CHANNEL_1, JogDirection.FORWARD)
+    device.move_absolute(
+        ChanIdent.CHANNEL_1, jog_count_1 * jog_step_1 * ureg.mpc320_step
+    )
```

Comment:
It would be good if we reset the `jog_step_1` param after running this.

After tomshen's PR to update how getting and setting params works is merged, we can do a `get_params` at the beginning of the test, store the value of `jog_count_1`, and then `set_params` at the end to reset it.

Even better, we can do this using `try` and `finally`, so that we can (almost) guarantee that the params value is reset even if something fails in the `jog` command.

Reply:
Yes, that sounds like a good idea. However, if I recall correctly, I remember you said something about not going too overboard with resetting the params of the machine, because there's a limited amount of writes or something. Or maybe I am misremembering.

Reply:
Since we run these tests manually, they will only be run a small number of times.

We don't know how the storage is implemented internally -- it could be an EEPROM or Flash storage -- and we don't know the lifetime of the storage. I've heard numbers ranging from 100,000 to 1,000,000 write cycles for EEPROM to much larger for Flash. We probably don't need to worry about it for now.

It might be that we should be implementing jog in software, in the end.

Reply:
Oh yes I see. Yeah, implementing it in software would probably be a good idea either way.